Create ui-c8y-1018-503-126-avoid-unnecessary-unavailability-alarms-for-lw-m-2-m-device-graft.md (#2566)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1018-503-126-avoid-unnecessary-unavailability-alarms-for-lw-m-2-m-device-graft.md

---
date: ""

> title: Avoid unnecessary unavailability alarms for LwM2M device GRAFT (#7392)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Avoid unnecessary unavailability alarms for LwM2M devices
```

> title: Avoid unnecessary unavailability alarms for LwM2M device GRAFT (#7392)

Reviewer: ```suggestion
title: Added new option to avoid unnecessary unavailability alarms for LWM2M devices
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3770
version: 1018.503.126
---

> Avoid unnecessary unavailability alarms for LwM2M device GRAFT (#7392)

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, LwM2M devices were incorrectly marked as unavailable, causing unnecessary unavailability alarms to be raised. This change fixes the issue by improving the device availability detection for LwM2M devices. With this fix, unavailability alarms will only be triggered for LwM2M devices when the device is actually unavailable, reducing noise and unnecessary alerts for users who are monitoring device availability.
```
